objects/Fan-MadeAccessories.aa8b38/CleanUpHelper.26cf4b.ttslua

```diff
@@ -80,88 +80,89 @@ buttonParameters.function_owner = self
 -- option loading and GUI setup
 ---------------------------------------------------------
 
-function onSave() return JSON.encode({ tidyPlayermats, importTrauma, resetResources }) end
+function onSave()
+  return JSON.encode({
+    options = options,
+    optionsVisible = optionsVisible
+  })
+end
 
-function onLoad(saved_data)
-  local loaded_data = JSON.decode(saved_data)
-  if loaded_data ~= nil then
-    tidyPlayermats = loaded_data[1]
-    importTrauma   = loaded_data[2]
-    resetResources = loaded_data[3]
+function onLoad(savedData)
+  if savedData ~= nil then
+    local loadedData = JSON.decode(savedData)
+    options = loadedData.options
+    optionsVisible = loadedData.optionsVisible or false
```

Comment:
Optional (heh) but I don't think the visibility of an options popup needs to carry across a save.  Should be fine to have it invisible on a fresh load.